SlackPrNotification.js

```diff
@@ -10,14 +10,16 @@ var authorName = process.env.PULL_REQUEST_AUTHOR_NAME;
 var authorIconUrl = process.env.PULL_REQUEST_AUTHOR_ICON_URL;
 var compareBranchOwner = process.env.PULL_REQUEST_COMPARE_BRANCH_OWNER;
 var compareBranchName = process.env.PULL_REQUEST_COMPARE_BRANCH_NAME;
+var baseRepoName = process.env.PULL_REQUEST_BASE_REPO_NAME;
 var baseBranchOwner = process.env.PULL_REQUEST_BASE_BRANCH_OWNER;
 var baseBranchName = process.env.PULL_REQUEST_BASE_BRANCH_NAME;
 var sendHereMention = process.env.IS_SEND_HERE_MENTION.toLowerCase() === "true" ? "<!here>\n" : "";
-var prFromFork = process.env.IS_PR_FROM_FORK;
-var compareBranchText = prFromFork === "true" ? compareBranchOwner + ":" + compareBranchName : compareBranchName;
-var baseBranchText = prFromFork === "true" ? baseBranchOwner + ":" + baseBranchName : baseBranchName;
 var makePretty = process.env.MAKE_PRETTY.toLowerCase() === "true"; //Priority is pretty > compact > normal
 var makeCompact = process.env.MAKE_COMPACT.toLowerCase() === "true";
+
+var baseBranchText = baseRepoName + ":" + baseBranchName;
```

Comment:
This is in a couple places, but I have two tidbits:

1. Personally, I think both base and compare should either have the owner and semi-colon prefix when it's relevant, or not at all; I'm not a huge fan of always printing the baseRepoName in the baseBranchText, but that could just be me and my pedanticness.
2. Why is the baseRepoName needed at all? Couldn't baseBranchOwner just be used and avoid the extra variable?

FYI I have not tested this change myself.

Reply:
baseBranchOwner is pointless in this case, since you are always going to find your own name/organization's name.

KingArrkinian/branch has no valuable information.
Repository/branch does however.

As for having the base repo name every time, again, I don't think the visual clutter is serious enough to warrant extra options on this.

More importantly, if we keep the IS_PR_FROM_FORK system, that means an extra option in the .yaml, which will in turn also probably have SHOW_TAGS, SHOW_USERS, etc.
Since more options will probably be added later on, I believe educated choices are more interesting than options for every  such item.

Reply:
Really it's more a question of whether we want educated choices or not. I just think it's better to not add extra booleans everywhere. Even with my own PR on the looks of things, so the MAKE_COMPACT and MAKE_PRETTY, I'd have probably taken out the original form of the message and made MAKE_PRETTY the standard with MAKE_COMPACT as sole option. But this isn't my repo so it's not my call :)

Reply:
Also maybe I should've said this from the start but I planned it this way in case an organisation has several repositories and one slack channel where they want the updates on PRs. Printing the repo name in Slack will be very important in case this happens.

Reply:
@KingArrkinian I see your point on the baseRepoName, and you answered my question about one Slack channel being used for several repositories before I had a chance to finish typing it haha. 

In my case, I try to avoid using one Slack channel for multiple repositories, thus why baseRepoName is irrelevant to me, but I see why it'd be useful for others. I don't see any easy way to combine ideologies without creating extra logic options, but I'm open to suggestions. However, after your explanation, I don't have any qualms; though as you said, I'm not the repo owner. :) Perhaps @jun3453 can weigh in regarding support and usefulness of Slack channels being one-to-many with repositories.

So, this change could be broken in two separate things:
1) Automating the fork part (this should definitely be done regardless)
2) Determining whether or not its worth supporting one-to-many with Slack and repos, respectively (which is mostly just displaying text differently)

Reply:
Agreed. I also tried at first to allow for keeping the option to show the repository/fork or not, but the more I tried to articulate a practical usage (the least unnecessary options with the most pertinent information given), the less I managed to agree with any option. In the end I just automated it and did this PR. :)

For the case of one channel taking multiple webhooks, I personally imagined the following workflow:

One slack channel per repo with pretty printing
And/or
One slack channel for all repositories with compact printing.

Both styles are completely valid, you pretty print on an occasional PR with the goal of an interactive usage (IE using Slack to look at the PR and communicate about it), or you compact print everything to one channel and use it only to have bookkeeping on your updates.
When you have dozens of updates over multiple repositories, having to quickly read through the messages is important, and being compact plays a lot into quickly reading up.

So @jun3453, please take the PR if you like :)

Reply:
One thing to mention now that I think on it though: 
Currently the syntax is
```
baseRepoName:branch
compareBranchOwner:branch
```

If we ever add tags, it might get modified.
```
baseRepoName/branch:tags
compareBranchOwner/branch:tags
```
For example.